src/lib/y2storage/device.rb

```diff
@@ -263,6 +263,29 @@ def potential_orphans
       []
     end
 
+    # Copies the device to a given devicegraph, connecting it to its corresponding parent
+    #
+    # @raise [RuntimeError] if the device has more than one parent
```

Comment:
This restriction made sense in the original scope of the method (`Y2Partitioner::BlkDeviceRestorer`). But here it looks hard to justify. Why a generic method called `Device#copy_to` has such restriction?

Reply:
To be honest, I didn't remember the reason of this restriction. I think we can drop the restriction off. I guess `Y2Partitioner::BlkDeviceRestorer` still has to avoid copying an object with several parents, hasn't it?

Reply:
Yes. We should not change the `BlkDeviceRestorer` behavior. And we should change the implementation of this (and test it) when there are several parents.

We can also rename this method to make very clear that is a limited copier.